### PR TITLE
Remove notions of old protocol from documentation

### PR DESCRIPTION
**Pull request by Patrik-Stas**
Signed-off-by: Patrik Stas <patrik.stas@absa.africa>

### CODE REVIEW

File: aries.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+* Connection:
```
Comment (mirgee) [inline aries.md:1]: Good, how about adding links to corresponding RFCs?

File: aries.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/credential.rs) 
+        * [State diagram](docs/states/aries/credential.puml) 
+
+* Credential Presentation:
+    * Verifier
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/proof.rs) 
+        * [State diagram](docs/states/aries/proof.puml) 
+    * Prover
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/disclosed_proof.rs) 
+        * [State diagram](docs/states/aries/disclosed-proof.puml)
```
Comment (mirgee) [inline aries.md:23]: We should consider generating `.png`s from the `.puml`s and updating them if necessary. Or removing them althogher, maybe links to the RFCs are sufficient? There is an overhead associated with upkeeping the diagrams, and they also change with versions.

Comment (Patrik-Stas) [reply aries.md:23]: I don't want to get into that now, just wanted to move this to reduce noise in readme.md

Comment (Patrik-Stas) [reply aries.md:23]: https://github.com/AbsaOSS/libvcx/issues/43

File: aries.md (hunk 1 above)
Comment (Patrik-Stas) [reply aries.md:1]: Added

**Pull request merged**